.storybook/main.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
 			(rule) => String(rule.test) === String(/\.(mjs|tsx?|jsx?)$/),
 		).exclude = nodeModulesExclude;
 
-		config.resolve.plugins ||= [];
+		config.resolve.plugins = [];
```

Comment:
Do we want to be changing this here?

Reply:
We do not! No idea how that got in there!